src/styles/utils/_loops.scss

```diff
@@ -0,0 +1,19 @@
+@for $index from 1 through 31 {
```

Comment:
Don't create a separate file for loops. They must be put with their selectors in one file